### ISSUE TITLE
Nothing appears in the output dir

### ISSUE THREAD

**Issue by jp7189** [Shondoit/lyne#2]
The script appears to run and process images, but nothing shows up in the output directory.  Below is my command and you can see 194 of 194 have been processed, but in 0 seconds which seems fishy to me.

C:\StableDiffusion\lyne-main\scripts>python add_weight_map.py --input-dir "C:\StableDiffusion\Training\photoset1" --output-dir "C:\StableDiffusion\Training\lyne-output" --mask-type attention --attention-prompt "a woman"
100%|███████████████████████████████████████████████████████████████████████████████| 194/194 [00:00<00:00, 265.75it/s]

**Comment by Shondoit** [comment]
I've updated the script so it now writes skip reasons in skip.txt in the output directory.
Can you run the script again and find out why the files are not processed?

**Comment by jp7189** [comment]
Thanks!  Super helpful.  I had tensorflow, but not pytorch.  After installing pytorch, everything is working great.

BTW, great work.  This util is going to streamline my workflow immensely.

For sake of completeness, here's what skip.txt had listed for every file:

CLIPSegForImageSegmentation requires the PyTorch library but it was not found in your environment.
However, we were able to find a TensorFlow installation. TensorFlow classes begin
with "TF", but are otherwise identically named to our PyTorch classes. This
means that the TF equivalent of the class you tried to import would be "TFCLIPSegForImageSegmentation".
If you want to use TensorFlow, please use TF classes instead!
